inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/yaml/KnowledgeBaseMapping.java

```diff
@@ -63,7 +67,8 @@
         @JsonProperty("label") String labelIri,
         @JsonProperty("property-type") String propertyTypeIri,
         @JsonProperty("property-label") String propertyLabelIri,
-        @JsonProperty("property-description") String propertyDescriptionIri)
+        @JsonProperty("property-description") String propertyDescriptionIri,
+        @JsonProperty("root-concept") List<String> rootConcepts)
```

Comment:
`root-concept` should be `root-concepts` here. Best introduce constants for the JSON property names.